src/condor_utils/ipv6_interface.cpp

```diff
@@ -42,6 +42,59 @@ uint32_t find_scope_id(const condor_sockaddr& addr) {
 	return result;
 }
 
+
+#else
+	// Win32
+
+#include <iphlpapi.h>
+
+uint32_t find_scope_id(const condor_sockaddr& addr) {
+	if (!addr.is_ipv6())
+		return 0;
+
+	ULONG family = AF_INET6;
+	ULONG flags = GAA_FLAG_INCLUDE_PREFIX; // Set the flags to pass to GetAdaptersAddresses
+	ULONG cbBuf = 16*1024; // start with 16Kb
+	PIP_ADAPTER_ADDRESSES pAddresses = nullptr;
+	auto_free_ptr addrbuf((char*)malloc(cbBuf));
+	int iterations = 0;
+
+	char ipbuf[IP_STRING_BUF_SIZE+1];
+	dprintf(D_ALWAYS, "Win32 find_scope_id(%s) id called\n", addr.to_ip_string(ipbuf, IP_STRING_BUF_SIZE, false));
```

Comment:
This probably shouldn't be D_ALWAYS, at least once we reason to believe this code works.

Reply:
Yeah, once we see it working it should be changed or removed.  the message is mainly there so we can ask if it's in the log if people report issues.